modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSKUs.testcase

```diff
@@ -158,6 +158,223 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-12218. Admin can added tier price on a product's SKU details page"
```

Comment:
This is a test for COMMERCE-12218. Admin can add tier price on a product's SKU details page